Gemfile

```diff
@@ -11,7 +11,8 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem 'voxpupuli-test', '>= 1.0.0',  :require => false
+  gem 'voxpupuli-test', '>= 1.4.0',  :require => false
+  gem 'puppet-lint-manifest_whitespace-check', :require => false
```

Comment:
I'd prefer to make this a part of vp-test 1.5.0 if it works well (which it appears to do)

Reply:
yes, me too. I just added it for testing / enable people to reproduce this.